### PR TITLE
DNN-8659 Change HostSettings and PortalSettings SettingValue column size

### PR DESCRIPTION
**Pull request by sleupold**
Adjust SettingValue column size for HostSettings, PortalSettings and RoleSettings and use same procedure construct for all settings tables.

### CODE REVIEW

File: Website/Providers/DataProviders/SqlDataProvider/08.01.00.01.SqlDataProvider [hunk 1]
```diff
@@ -166,3 +162,389 @@ AS
...
+GO
+
+CREATE PROCEDURE {databaseOwner}[{objectQualifier}UpdateHostSetting]
+	@SettingName	   nvarchar( 50),
+	@SettingValue	   nvarchar(Max),
+	@SettingIsSecure   bit,
+	@UserID	           int
+AS
+BEGIN
+	UPDATE {databaseOwner}[{objectQualifier}HostSettings]
```
Comment (bdukes) [inline Website/Providers/DataProviders/SqlDataProvider/08.01.00.01.SqlDataProvider]: Doesn't the `MERGE` below replace this `UPDATE`?

File: DNN Platform/Library/Data/DataProvider.cs [hunk 2]
```diff
@@ -451,12 +451,6 @@ public virtual string GetUnappliedIterations(string version)
 
 		#region Host Settings Methods
 
-		public virtual void AddHostSetting(string SettingName, string SettingValue, bool SettingIsSecure,
```
Comment (bdukes) [inline DNN Platform/Library/Data/DataProvider.cs]: Is there not a backwards compatibility concern with removing these methods (and their corresponding stored procedures)?

File: Website/Providers/DataProviders/SqlDataProvider/08.01.00.01.SqlDataProvider (hunk 1 above)
Comment (sleupold) [inline Website/Providers/DataProviders/SqlDataProvider/08.01.00.01.SqlDataProvider]: you are right, I used the update to update the correct column and forgot to remove the statement

File: DNN Platform/Library/Data/DataProvider.cs (hunk 2 above)
Comment (sleupold) [inline DNN Platform/Library/Data/DataProvider.cs]: Brian, 
the stored procedures are not part of the API, nor the DataProvider methods.
Only the controllers and objects, which still use same signatures and functionality.

**Comment by sleupold** [comment]
changes should be complete now. Using same functionality and patterns for all settings tables (Site Settings is a bit ahead, as it already supports multiple languages).

**Pull request merged**